app/services/dual-output/dual-output.ts

```diff
@@ -570,6 +633,16 @@ export class DualOutputService extends PersistentStatefulService<IDualOutputServ
     this.SET_IS_LOADING(status);
   }
 
+  /**
+   * Part of a workaround to set recording quality on the frontend to avoid a backend bug
+   * @remark TODO: remove when migrate output and stream settings to new API
+   * @param mode - single or dual output
+   * @param quality - recording quality
+   */
+  setDualOutputRecordingQuality(mode: 'single' | 'dual', quality: ERecordingQuality) {
```

Comment:
Could we extract `'single' | 'dual'` to a type so we can use it below (and probably in other places)?

Reply:
Done